aten/src/ATen/native/Pooling.cpp

```diff
@@ -22,14 +22,14 @@ static void check1d(
 }
 
 Tensor adaptive_avg_pool1d(const Tensor & self, IntArrayRef output_size) {
-  checkDim("adaptive_avg_pool1d", TensorArg(self, "self", 1), 3);
+  checkDimRange("adaptive_avg_pool1d", TensorArg(self, "self", 1), 2, 4);
```

Comment:
In other parts of the codebase, it's common to add a comment `/* exclusive */` at the end of this call to `checkDimRange()` to make it clear (see [here](https://github.com/pytorch/pytorch/blob/5da507b57bee31805217889b9f8ff673915dbc3a/aten/src/ATen/native/cudnn/BatchNorm.cpp#L82)).